changes/change.go

```diff
@@ -0,0 +1,107 @@
+package changes
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+)
+
+// ChangeTemplate is an editor friendly template for creating or modifying a Change.
+const ChangeTemplate = `modules: %s
+# change type is one of: feature, bugfix
+change type: %s
+description: %s`
+
+// Change represents a change to one or more Go modules.
+type Change struct {
+	Id            string
+	SchemaVersion string
+	Modules       []string
+	Type          string
+	Description   string
+	SetVersion    string
+}
+
+func NewChange(modules []string, changeType, description string) *Change {
```

Comment:
For each module targeted for a change, would it be simpler to write out a change description for each module. This simplifies later discovery of pending changes for an individual module. That's not to say you couldn't have the ability to write a changelog as user that targets multiple modules, but when stored in the repository would it make more sense to write out a file per module?